### PR TITLE
created mentor's dashboard, profile, home page; still working on disp…

### PR DESCRIPTION
**Pull request by berdiyeva**
…laying mentor's information; created mentorReducer file.

### CODE REVIEW

File: src/components/pages/Mentor/MentorProfile.js [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
-const MentorProfile = () => {
-  return <div>Hello!</div>;
-};
+const MentorProfile = ({ fetchMentors, profile }) => {
+  useEffect(() => {
+    fetchMentors(1); // change this later with login
+  }, [fetchMentors]);
 
-export default MentorProfile;
+  console.log('inside the mentor edit form', profile);
```
Comment (Chris-Forrest) [inline src/components/pages/Mentor/MentorProfile.js:10]: These are the only two problems I found with the code

File: src/components/pages/Mentor/MentorDashboard.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  LogoutOutlined,
+} from '@ant-design/icons';
+
+const MentorDashboard = props => {
+  const { profile } = props;
+
+  useEffect(() => {
+    props.fetchMentors(1); // change this later with login
+  }, []);
+  console.log('Mentors:', profile);
```
Comment (Chris-Forrest) [inline src/components/pages/Mentor/MentorDashboard.js:25]: Just a reminder they want us to remove all of our console.log calls that we used for testing purposes so the calls aren't in production code.

**Review by Chris-Forrest — APPROVED** [review]
Your code looks good and seems to follow the previous team's styling guide. My only criticism is that the Lambda reviewers would like us to get in the habit of doing smaller amounts of code in our pull requests

File: src/components/pages/Mentor/MentorDashboard.js (hunk 2 above)
Comment (berdiyeva) [reply src/components/pages/Mentor/MentorDashboard.js:25]: Thanks for the reminder I will remove them.

File: src/components/pages/Mentor/MentorHome.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <div>
+      <h1 style={{ textAlign: 'center' }} className="page-title">
+        Mentor Home Page
+      </h1>
+    </div>
+  );
+};
+
+const mapStateToProps = state => {
```
Comment (austinmehmet) [inline src/components/pages/Mentor/MentorHome.js:23]: Not something you need to change but, did you know that you do not need a return statement if you are just returning an object literal. There is just 1 trick you need to know to do it, wrap the object in parentheses.

```js
const someFunc= () => ({
  dogs: true,
  cats: false
})
```

```js
const mapStateToProps = state => ({
  profile: state.mentorReducer.mentorProfile
})
```

Useful if you like the no return syntax allowed by arrow functions that are one-liners

File: src/components/pages/Mentor/MentorDashboard.js (hunk 2 above)
Comment (jtwillo51) [reply src/components/pages/Mentor/MentorDashboard.js:25]: I was about to say the same thing, got me too lol

**Review by jtwillo51 — APPROVED** [review]

File: src/components/pages/Mentor/MentorHome.js (hunk 3 above)
Comment (berdiyeva) [reply src/components/pages/Mentor/MentorHome.js:23]: Now I know that, thank you for sharing.

**Pull request merged**